src/sentry/data/config/sentry.conf.py.default

```diff
@@ -14,6 +14,8 @@ DATABASES = {
         'PASSWORD': '',
         'HOST': '',
         'PORT': '',
+        'AUTOCOMMIT': True,
```

Comment:
I guess it doesn't matter. These seem to be the default from Django, we're just declaring these values explicitly.